package.json

```diff
@@ -4,6 +4,7 @@
     "start": "npm-run-all deploy:fork:reset initialize",
     "prepare": "husky install",
     "test": "npm run deploy:fork:reset && hardhat --network mainnetFork test:dir --deploy-fixture ./test/integration/",
+    "test-no-deploy": "hardhat --network mainnetFork test:dir ./test/no-deploy/",
```

Comment:
Can you elaborate a bit on what exactly is that? Why we needed it?

Reply:
It runs a subset of tests without deploying the whole system. It allows to test a smart contract in isolation. Arguably it can be called unit tests. It runs probably 100 times faster if you just want to run a single test.

Reply:
Go it!
So yeah, let's change the name of the script / paths to match unit tests instead..

I'll then add it to the CI